web/src/app/modules/views/side/modules/errors-warnings/components/warning.component.css

```diff
@@ -0,0 +1,17 @@
+.warning-link {
```

Comment:
We can use normal links for the warning-items and just disable the navigation by stopping the event propagation and disable the default action. This way, the links would look more natural in specmate.